lib/ts/ingredients/emaildelivery/services/smtp.ts

```diff
@@ -22,6 +22,7 @@ export interface SMTPServiceConfig {
     };
     port: number;
     secure?: boolean;
+    username: string;
```

Comment:
Consider making this optional and if not provided, we fallback on using the `from.email`

Reply:
Done